### DISCUSSION TITLE
非上标式引用文献编号？

### DISCUSSION THREAD

**Discussion by iTomxy** [sikouhjw/gdutthesis#54] [Q&A] [answered]
`\gdutcite` 和 `\cite` 都是上标形式的，形如“…成果<sup>[1]</sup>”；但当直接提及引用文献时需要**非**上标的格式，即“由文献[8, 10～14]可知”[^1]。此时应该用什么命令呢？

[^1]: https://yjs.gdut.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1533825352&wbfileid=3799073

**Comment by xkwxdyy** [comment]
如果是 `biblatex` 方式的话，可以试试 `\parencite`

**Reply by iTomxy** [reply]
我用的编译命令是[这个](https://github.com/sikouhjw/gdutthesis/discussions/24#discussion-3929924)，即 xelatex + bibtex + xelatex + xelatex，用 `\parencite` 会变成 `文献 aaai14scm 提出…`，其中 `aaai14scm` 是我在 bibtex 条目里给它起的那个名字。

**Reply by sikouhjw** [reply]
> 我用的编译命令是[这个](https://github.com/sikouhjw/gdutthesis/discussions/24#discussion-3929924)，即 xelatex + bibtex + xelatex + xelatex，用 `\parencite` 会变成 `文献 aaai14scm 提出…`，其中 `aaai14scm` 是我在 bibtex 条目里给它起的那个名字。

所以你用的是 `bibtex`，不能用 `biblatex` 模式的方法。

**Comment by sikouhjw** [comment, marked as answer]
`\gdutcite*` 是非上标引用，我在 `main.tex` 里写了例子。

主要问题是手册是一个发行版发布一次，不便于用户查看，我觉得没有太好的办法。

如果不是『最新的（非发行版）』版本，可以在导言区加上
```
\newcommand{\mycite}[1]{\setcitestyle{numbers}\cite{#1}\setcitestyle{super}}
```
就可以非上标引用了。（仅限 `bibtex`，如果是 `biblatex`，像 @xkwxdyy 说的一样用 `\parencite` 就可以了）
